scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -612,181 +611,176 @@ scater::plotReducedDim(merged_sce,
   ggtitle("UMAP after integration with fastMNN")
 ```
 
-This `fastMNN_UMAP` certainly looks different from the one we made from `merged_UMAP`!
+This `fastmnn_UMAP` certainly looks different from the one we made from `merged_UMAP`!
+What different trends do you see? 
+Do all samples look "equally well" integrated, from a first look?
 
+Importantly, one reason that batches may still appear separated in the corrected UMAP is if they _should_ be separated - for example, maybe two batches contain very different cell types, have very different diagnoses, or may be from different patients.
 
-However, it's a bit challenging to see all the points given the amount of overlap in the plot.
-One way we can see all the points a bit better is to facet the plot by library:
-
-```{r fastmnn umap batches faceted}
-scater::plotReducedDim(merged_sce,
-                       dimred = "fastMNN_UMAP",
-                       colour_by = "library",
-                       point_size = 0.5,
-                       point_alpha = 0.2, 
-                       # Allow for faceting by a variable using `other_fields`:
-                       other_fields = "library") +
-  guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
-  ggtitle("UMAP after integration with fastMNN") +
-  facet_wrap(~library)
-```
+Recall from earlier that we conveniently have cell type annotations in our SCEs, so we can explore those here!
+Let's take a quick detour to see what kinds of cell types are in this data by making a barplot of the cell types across samples:
 
+```{r explore celltypes}
+# Cell types are stored in the `celltype_broad` and `celltype_fine` columns in the SCE
+merged_sce_df <-  as.data.frame(colData(merged_sce)) 
 
+ggplot(merged_sce_df, 
+       aes(x = sample, 
+           fill = celltype_broad)
+) +
+  # Barplot of celltype counts, arranged horizontally
+  geom_bar(position = "fill") + 
+  # Use a CVD-friendly color scheme
+  scale_fill_brewer(palette = "Dark2", na.value = "grey80") +
+  # nicer theme
+  theme_bw() 
+```
+We see that Tumor cell types are by far the most prevalent across all samples, and normal tissue cell types are not very common.
```

Comment:
````suggestion
```

We see that Tumor cell types are by far the most prevalent across all samples, and normal tissue cell types are not very common.
````